src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,120 @@
-export const Pagination = () => {};
+import { useState } from 'react';
+
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  currentPage: number,
+  perPage: number,
+  onPageChange: (value: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage, perPage, onPageChange, total,
+}) => {
+  const [items] = useState(getNumbers(1, total));
+  const lastPage = Math.ceil(items.length / perPage);
+
+  const handleClick = (page: number) => {
+    if (page === currentPage) {
+      return;
+    }
+
+    onPageChange(page);
+  };
+
+  const displayPagination = Array.from({ length: lastPage }, (_, i) => i + 1)
+    .map(page => (
+      <li
+        key={page}
+        className={
+          cn(
+            'page-item',
+            { active: currentPage === page },
+          )
+        }
+      >
+        <a
+          href={`#${page}`}
+          data-cy="pageLink"
+          className="page-link"
+          onClick={() => handleClick(page)}
+        >
+          {page}
+        </a>
+      </li>
+    ));
+
+  const filteredItems = (
+    itemsList: number[], forPage: number, currPage: number,
+  ) => {
```

Comment:
Use vertical formatting if you have > 2 params
```suggestion
  const filteredItems = (
    itemsList: number[],
    forPage: number,
    currPage: number,
  ) => {
```